### DISCUSSION TITLE
Blurry text in web demo in Chrome but not in Firefox

### DISCUSSION THREAD

**Discussion by nikitablack** [emilk/egui#129] [General]
Hello, I wonder what can cause such a behavior? I have two browsers opened and see the crisp text in Firefox and a very VERY blurry text in Chrome. Also, I noticed that the blurriness changes when I draw/resize a chrome window while in Firefox it always stays sharp.

OS: Ubuntu 18.04, no monitor scale, browsers zoom is at 100%/

![Screenshot from 2021-01-21 17-52-53](https://user-images.githubusercontent.com/6390092/105383784-c7ba3200-5c11-11eb-96d3-8652768131af.png)

**Comment by emilk** [comment]
It sounds like this is due to https://github.com/emilk/egui/issues/103

**Comment by nikitablack** [comment]
Oh, I see. I searched in issues but somehow missed it. So there's no solution yet?

**Reply by emilk** [reply]
Fixed in https://github.com/emilk/egui/pull/151 !

**Reply by nikitablack** [reply]
Unfortunately, I still see the blurry text in Chrome in the demo.

**Reply by emilk** [reply]
I haven't published new demo yet. Will do when 0.9 is released.

**Reply by zicklag** [reply]
0.9 looks great for me in Chrome. :smiley:
